lib/network/Images.js

```diff
@@ -1,4 +1,4 @@
-import CachedImage from './CachedImage';
+var CachedImage = require('./CachedImage').default;
```

Comment:
i'll check for these occurrences. For mixed imports I'll modify the export to CJS style, for unmixed imports i'll revert to ES6 exports.